staging/src/k8s.io/legacy-cloud-providers/gce/gce_instances.go

```diff
@@ -426,6 +426,18 @@ func (g *Cloud) AddAliasToInstance(nodeName types.NodeName, alias *net.IPNet) er
 // Gets the named instances, returning cloudprovider.InstanceNotFound if any
 // instance is not found
 func (g *Cloud) getInstancesByNames(names []string) ([]*gceInstance, error) {
+	foundInstances, err := g.getFoundInstanceByNames(names)
+	if err != nil {
+		return nil, err
+	} else if len(foundInstances) != len(names) {
```

Comment:
Remove "else" - the smaller branching in the code is, the easier to reason about (and you have return in if branch anyway).
So:
```
if err != nil {
  return nil, err
}
if len(found...
```

Reply:
ack